app/views/api_docs/reference/v0.1/_reference.md

```diff
@@ -1233,7 +1233,9 @@ Withdraw a trainee.
 
 #### Request
 
-`POST /api/v0.1/trainees/{trainee_id}/withdraw`
+`POST /api/v0.1/trainees/{trainee_id}/withdraw?reasons[]={reasons}&reasons[]={reasons}&withdraw_date={withdraw_date}&withdraw_reasons_details={withdraw_reasons_details}&withdraw_reasons_dfe_details={withdraw_reasons_dfe_details}`
```

Comment:
```suggestion
`POST /api/v0.1/trainees/{trainee_id}/withdraw?reasons[]={reasons}&withdraw_date={withdraw_date}&withdraw_reasons_details={withdraw_reasons_details}&withdraw_reasons_dfe_details={withdraw_reasons_dfe_details}`
```

Reply:
Yeah, it looks a bit odd repeated in this context, but you're right it might not be clear how to pass in multiple `reasons`, as there are [many different ways](https://swagger.io/docs/specification/2-0/describing-parameters/#array) to pass array values in query params.